iotic-spacectl/iotic-spacectl-intro/step3.md

```diff
@@ -1,17 +1,24 @@
 # Model a feed
 
-As we saw in the first step, twins can have metadata associated with them - comments, labels, tags etc.
+As we saw in step 1, twins can have metadata associated with them - comments, labels, tags etc.
+
 Twins can also have feeds for real time data.
+
 We've already created a twin of our house, so lets look at how to add a feed.
+
 As people come and go from our house, the number occupants changes. Lets model this as a feed on our twin.
 
 ## Model the feed
 
-If you want to skip this step, we included a ready modeled feed, so you can execute `cp example-feed.json feed.json`{{execute}} and skip to 'create the feed on our house twin'.
+If you want to skip this step, we've included a ready modelled feed, just run the code in the below drop down and skip to the next 'create the feed on our house twin':
 
+<details>
+<summary>Skip this step</summary>
+<br>
 `./iotic-spacectl modelfeed`{{execute}}
```

Comment:
This should be the cp command shouldn't it?
eg to skip this step do `cp example-feed.json feed.json`

Then for everyone else `./iotic-spacectl modelfeed`

Reply:
You're right, updated.